README.md

```diff
@@ -1 +1,19 @@
 # citation-plugin
+
+citation-plugin is a firefox plugin which presents the [CitationCore](https://github.com/mozillascience/CitationCore) library in a user friendly way in browswer. It is part of a larger effort, lead by Mozilla Science Lab, to develop a suite of tools that aim to implement a standard for citing software and making it easier to cite software correctly.  To learn more about this project you can visit the [Software Citation Tools repository](https://github.com/mozillascience/software-citation-tools).
+
+## Install
+To install:
+```
+cd Plugin
+npm install 
+```
+
+Once the install script finishes, navigate to [about:debugging#addons](about:debugging#addons). Once there, select the the "Load Temporary Add-on", navigate to the Plugin directory and select the manifest.json and click load
```

Comment:
In 'select the the "Load Temporary Add-On"', "the" is repeated twice. However, it isn't followed up by "button" or "link" or the like, and the article should probably be removed all together, changing it to 'select "Load Temporary Add-On"'